packages/amplify-provider-awscloudformation/lib/initializer.js

```diff
@@ -14,9 +14,10 @@ async function run(context) {
   await configurationManager.init(context);
   if (!context.exeInfo || context.exeInfo.isNewEnv) {
     const initTemplateFilePath = path.join(__dirname, 'rootStackTemplate.json');
-    const timeStamp = `${moment().format('YYYYMMDDHHmmss')}`;
+    let timeStamp = `${moment().format('YYYYMMDDHHmmss')}`;
+    timeStamp = timeStamp.slice(timeStamp.length - 5);
```

Comment:
This leads to the timestamp to be 'Hmmss', I don't think that's what we wanted.  If you want to reduce the length, you can directly do it on the variable definition `${moment().format('.....')}`